app/res/layout/setup_install.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                xmlns:SquareButtonWithText="http://schemas.android.com/apk/res/org.commcare.dalvik"
+                android:id="@+id/screen_first_start_main"
+                android:layout_width="fill_parent"
+                android:layout_height="fill_parent"
+                android:orientation="vertical"
+                android:background="@color/cc_core_bg"
```

Comment:
We can probably set this in the Theme, btw, so we don't need to set it on each layout host.